components/bottom_sheets/PersistentBottomSheet.vue

```diff
@@ -1,6 +1,13 @@
 <template>
     <div class="text-center">
-        <v-bottom-sheet v-model="sheet" persistent>
+        <v-bottom-sheet
+            v-model="sheet"
+            persistent
+            overlay-color="purple"
+            overlay-opacity="0.1"
+            width = 500
+            no-click-animation
```

Comment:
Could you tell me please what property to check that on site?

Reply:
It is **Persistent bottom sheet**

Reply:
I am about "no-click-animation"

Reply:
Oh, I see. When we click outside bottomsheet the layout 'bounces', with 'no-click-animation' feature it stands still.

Reply:
feature 'no-click-animation' - not testable - deletion approved by @omega790 (info in matrix)